### PR TITLE
CLOUD-2477 updates for consolidated modules

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by rcernich** [comment]
@errantepiphany, @spolti, please merge this.  The related changes in cct_module have been merged.

File: image.yaml [hunk 1]
```diff
@@ -147,8 +148,6 @@ modules:
           - name: os.kieserver.s2i
           - name: os.kieserver.webapp
           - name: os.kieserver.chmod
-          - name: os-java-run
```
Comment (spolti) [inline image.yaml:150]: Hey Rob, should we also remove these two modules from other images? Like here for example:
https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/rhpam71-dev/businesscentral/image.yaml#L131

Comment (rcernich) [reply image.yaml:150]: You'd have to test those changes.  Only the changes in the PR have been tested by QE.  That said, I believe all that module does now is redirect to jboss.container.java.jvm.bash, which includes the jvm gc and proxy related configuration scripts.  Those should also be included through one of the maven s2i, but you'd have to verify the build, etc.  All I did was refactor the core cct_modules.  I did not migrate any of the images, other than openjdk.  (This also helped prove that the changes were backwards compatible.)

Comment (errantepiphany) [reply image.yaml:150]: @spolti, @rcernich:

We can merge stuff into master for >= CE sprint-24 if it changes ONLY os.kieserver.* modules in that repo, which are for bpms/brms 6.x images.

For our rhdm/rhpam 7.x images, we only use the jboss-kie-* modules in that repo, but we should not be merging anything into there that won't work for our 7.1 release, which is tied to CE sprint-23. If the changes Rob made are only in CE sprint-24, we should not be changing our jboss-kie-* module scripts, or any rhdm or rhpam image.yaml files for the 7.1 release.

**Comment by errantepiphany** [comment]
It looks like this PR is safe to merge, since it only affects os.kieserver.*  modules. If it touched jboss-kie-* modules, then we wouldn't be able to merge it until after 7.1 is released, at which point we'll move from CE sprint-23 to CE sprint-24. @spolti , do you agree?

File: image.yaml (hunk 1 above)
Comment (spolti) [reply image.yaml:150]: Thanks @rcernich.

**Comment by spolti** [comment]
@errantepiphany I do agree.

**Review by spolti — APPROVED** [review]

**Pull request merged**